snuba/web/rpc/v1/endpoint_get_trace.py

```diff
@@ -57,7 +66,127 @@
 }
 
 
-def _build_query(request: GetTraceRequest, item: GetTraceRequest.TraceItem) -> Query:
+def get_pagination_max_items() -> int | None:
+    """
+    returns an integer > 0, or None if there is no limit. This is configured using the snuba setting ENDPOINT_GET_TRACE_PAGINATION_MAX_ITEMS
+    """
+    if (
+        ENDPOINT_GET_TRACE_PAGINATION_MAX_ITEMS is None
+        or ENDPOINT_GET_TRACE_PAGINATION_MAX_ITEMS == 0
+    ):
+        return None
+    if ENDPOINT_GET_TRACE_PAGINATION_MAX_ITEMS < 0:
+        # warning that the pagination max items is negative
+        sentry_sdk.capture_message(
+            f"Pagination max items is negative: {ENDPOINT_GET_TRACE_PAGINATION_MAX_ITEMS}"
+        )
+        return None
+    return ENDPOINT_GET_TRACE_PAGINATION_MAX_ITEMS
+
+
+# positive integer or None, representing no limit
+PAGINATION_MAX_ITEMS: int | None = get_pagination_max_items()
+
+
+class EndpointGetTrace_PageToken:
```

Comment:
Is there some kind of namespace conflict with protobuf-generated classes? Because in general you don't need to add underscores into camel-case names